CONTRIBUTING.md

```diff
@@ -227,24 +227,44 @@ To start debugging JHipster with **VSCode**, open the generator code in your wor
 
 It is also possible to debug sub generators by selecting one of the other debug options (for example `jhipster entity`). Those debug configurations are specified in the `.vscode/launch.json` file.
 
-## Local Travis Build
+## Local Travis Build and explore application generated
 
-You can run the travis builds locally by following below commands
+You can run the Travis builds locally by following below commands, and track rendered pages.
 
-CD into the travis folder `cd travis` from the generator source code root folder
+#### How to use it
 
-Run `./build-samples.sh [command_name] [sample_name:optional]`
+1. Change directory into the [./travis](./travis) folder `cd travis` from the generator source code root folder
 
-This will create the travis sample project under the `travis/samples` folder with folder name `[sample_name]-sample`. You can open this application in your editor or IDE to check it further. You can also run tests locally on the project to verify
+2. Do not forget to run `yarn link`.
```

Comment:
This comment here might be misleading. `yarn link` should be run on the generator source code root folder

Reply:
No, as almost every `yarn` command, `yarn` « walked up the directory tree to find the project root and run the command from there ». This behaviour is very cool for Neovim users ;-). 

It's that why we must trigger in ./build-samples.sh something like : `cd samples/sample_name-sample && yarn init -y && yarn link "generator-jhipster"`.